slick/src/main/scala/slick/jdbc/StaticQuery.scala

```diff
@@ -91,10 +88,14 @@ object ActionBasedSQLInterpolation {
 }
 
 case class SQLActionBuilder(queryParts: Seq[Any], unitPConv: SetParameter[Unit]) {
+
+  def ++(that: SQLActionBuilder): SQLActionBuilder =
+    SQLActionBuilder(this.queryParts ++ that.queryParts, SetParameter.compose(this.unitPConv, that.unitPConv))
+
   def as[R](implicit rconv: GetResult[R]): SqlStreamingAction[Vector[R], R, Effect] = {
     val query =
-      if(queryParts.length == 1 && queryParts(0).isInstanceOf[String]) queryParts(0).asInstanceOf[String]
-      else queryParts.iterator.map(String.valueOf).mkString
+      if(queryParts.length == 1 && queryParts(0).isInstanceOf[String]) queryParts(0).asInstanceOf[String].trim
+      else queryParts.iterator.map(String.valueOf).mkString.trim
```

Comment:
Maybe I should just remove the extra space the macro adds at the end. On the other hand, we could just let it be, does not really affect much does it?